app/views/openid_connect/shared/redirect_js.html.erb

```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
-<html>
+<html class="no-js">
```

Comment:
It'd be nice to have a base-ier [base template](https://github.com/18F/identity-idp/blob/main/app/views/layouts/base.html.erb) that we could extend, to avoid having to bring in these individual elements.

There's probably some other features from the base template we want here too, like the `lang` attribute, some of the meta tags, etc.